### DISCUSSION TITLE
Welcome to Azure PowerShell Administration Discussions!

### DISCUSSION THREAD

**Discussion by tomwechsler** [tomwechsler/Azure_PowerShell_Administration#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JuergenBelle** [comment]
Hi Tom, I'm System Administrator at Wieland Group. Mainly I do Software Deployment. I hope some more people will join to discuss all about Azure Powershell and Automation Solutions with Powershell.
